docs/imagecustomizer/api/configuration/outputSelinuxPolicy.md

```diff
@@ -0,0 +1,62 @@
+---
+parent: Configuration
+ancestor: Image Customizer
+---
+
+# outputSelinuxPolicy type
```

Comment:
Delete this doc. `outputSelinuxPolicy` is not a struct in the API. Instead, all the documentation should be under `output.md`.